cloudrun-malware-scanner/deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Check if the environment value is provided as a command-line argument
+if [ $# -ne 3 ]; then
+    echo -e "Usage: $0 <env> <service_name> <service_account_name>"
+    echo -e "Possible values for <env> are: [pre-qa, qa, stg, prod]"
+    exit 1
+fi
+
+# Set the environment value from the command-line argument
+env_value="$1"
+service_name="$2"
```

Comment:
Would be nice to get a version of the service as an arg but I'm not sure it's entirely necessary.  eg: `cloudrun-malware-scanner-1.2.0` or whatever.

Reply:
actually you cant have `.` in the service name so I'm going to skip this for now but would like to circle back to this later